src/kubeobjects/execution_env.go

```diff
@@ -0,0 +1,7 @@
+package kubeobjects
+
+import "os"
+
+func IsRunLocally() bool {
```

Comment:
`kubesystem` might be a better location for that function since it does not interact with any objects, only the environment